hosted/Dockerfile

```diff
@@ -32,6 +31,43 @@ WORKDIR server/eb
 RUN make build-ecs
 WORKDIR /
 
+#nginx config
+ENV NGINX_VERSION=1.21.6
+RUN wget  http://nginx.org/download/nginx-$NGINX_VERSION.tar.gz && \
+    tar -zxvf nginx-$NGINX_VERSION.tar.gz && \
+    rm nginx-$NGINX_VERSION.tar.gz && \
+    cd nginx-$NGINX_VERSION && \
+    ./configure --prefix=/etc/nginx \
+                --sbin-path=/usr/sbin/nginx \
+                --conf-path=/etc/nginx/nginx.conf \
+                --http-log-path=/var/log/nginx/access.log \
+                --error-log-path=/var/log/nginx/error.log \
+                --with-pcre  --lock-path=/var/lock/nginx.lock \
+                --pid-path=/var/run/nginx.pid \
+                --modules-path=/etc/nginx/modules && \
+    make && \
+    make install
+
+RUN git clone https://github.com/eustas/ngx_brotli.git && \
+    cd ngx_brotli && git submodule update --init && cd ../
+
+RUN cd nginx-$NGINX_VERSION && \
+    ./configure $(echo $(nginx -V 2>&1) | awk -F':' '{print $3}' | awk '{$1=$1;print}') --add-dynamic-module=../ngx_brotli && \
+    make modules
+
+RUN mkdir /etc/nginx/modules && \
+    mkdir /etc/nginx/sites-available && \
+    mkdir /etc/nginx/sites-enabled && \
+    cd nginx-$NGINX_VERSION && \
+    cp objs/ngx_http_brotli_filter_module.so /etc/nginx/modules/ngx_http_brotli_filter_module.so && \
+    cp objs/ngx_http_brotli_static_module.so /etc/nginx/modules/ngx_http_brotli_static_module.so && \
+    chmod 644 /etc/nginx/modules/ngx_http_brotli_filter_module.so && \
+    chmod 644 /etc/nginx/modules/ngx_http_brotli_static_module.so
```

Comment:
is it reasonable to place all of the nginx build cmds into a script and just call it? if only to reduce the visual complexity of the dockerfile.